versioned_docs/version-22.10/graph-views/map-web-upgrade.md

```diff
@@ -72,11 +103,22 @@ sudo yum update centreon-map-engine centreon-map-web-client
 </TabItem>
 </Tabs>
 
-2. Make sure you are using the correct version of MariaDB and update it if needed. See [Upgrading MariaDB](../upgrade/upgrade-mariadb.md).
+3. Clear your browser cache.
+
+4. Finalize the update of the module and the widget in the Centreon interface **Administration > Extensions > Manager**.
+
+ > An orange update button is visible, indicating that an update is available. Click on it to update the module, and do the same for the widget.
 
-## Update through the interface
+5. Restart the **centreon-map-engine** service using the following command:
+ 
+  ```shell
+  sudo systemctl start centreon-map-engine
+  ```
+
+6. If you are using MAP Legacy, enter this command on the MAP server to apply any customization.
 
-Log on to the Centreon web interface and go to `Administration > Extension >
-Manager`.
+ ```shell
+ diff -u /etc/centreon-studio/centreon-map.conf /etc/centreon-studio/centreon-map.conf.rpmsave
```

Comment:
This command only works for EL7/EL8/EL9 but not on Debian.
Debian will not create .rpmsave files
Also, if we update Centreon Map, we don't care about /etc/centreon-studio/ folder because it's related to centreon Map Legacy and the update will not modify these files